tsconfig.json

```diff
@@ -4,7 +4,7 @@
         "module": "commonjs",
         "lib": [
             "dom",
-            "es2017",
+            "esnext",
```

Comment:
Same as lint rule change, make sense to have a separated PR for this. There you can explain the reason behind these changes and we can focus discussing and reviewing these changes.